bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/api/Enums.java

```diff
@@ -117,4 +117,23 @@ public static FanMovement fromValue(int value) {
             return STOPPED;
         }
     }
+
+    public enum HomekitMode {
+        AUTO            ("on"),
```

Comment:
Oops, forgot about this one. Looks like this would be the last change
```suggestion
        AUTO            ("auto"),
```

Reply:
Fixed this one now too